test/src/test/integration/link_git_protocol.rs

```diff
@@ -317,13 +335,17 @@ where
         assert!(out.pack.is_some());
     }
 
-    let remote_repo = git2::Repository::open(&remote).unwrap();
+    let mut remote_repo = git::open(remote.clone()).unwrap().into_easy_arc_exclusive(); // without GATs we need the arc version
```

Comment:
GATs would be needed on gitoxide's side?

Reply:
Yes, just there. [Here is the code for that](https://github.com/Byron/gitoxide/blob/7c559d6e1b68bc89220bca426257f383bce586ae/git-repository/src/easy/impls.rs#L72-L88) and the reason mutable `RefMut<'argh_a_lifetime_that_cannot_be_expressed, Repository>` can't be done yet.
```suggestion
    let mut remote_repo = git::open(remote.clone()).unwrap().into_easy_arc_exclusive(); // without GATs we need the arc version
```
```suggestion
    let mut remote_repo = git::open(remote.clone()).unwrap().into_easy_arc_exclusive(); // without GATs we need the arc version
```